pkg/dbmate/db_116.go

```diff
@@ -0,0 +1,59 @@
+// +build go1.16
```

Comment:
I'm not particularly attached to retaining go v1.15 support, since dbmate is primarily a standalone tool we don't provide explicit backwards compatibility promises when used as a library. Anyone stuck on earlier versions of Go can continue using the current version of dbmate until they are ready to upgrade.